### PR TITLE
letter_opener made easy

### PR DESCRIPTION
**Pull request by akostadinov**
**What this PR does / why we need it**:

Makes easy to enable `letter_opener` by a simple environment variable at runtime.

```
$ LETTER_OPENER=1 bundle exec sidekiq ...
```

**Which issue(s) this PR fixes** 

No issue.

**Verification steps** 

- export LETTER_OPENER=1
- bundle exec sidekiq ...
- in Porta web UI invire a user
- a browser tab should open with email content present

**Special notes for your reviewer**:

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -57,8 +57,14 @@
   config.action_dispatch.best_standards_support = :builtin
 
   config.action_mailer.perform_deliveries = true
-  # See emails in browser
-  config.action_mailer.delivery_method = defined?(LetterOpener) ? :letter_opener : :test
+
+  if defined?(LetterOpener)
+    config.action_mailer.delivery_method = :letter_opener
+  elsif config.action_mailer.smtp_settings[:address]
+    config.action_mailer.delivery_method = :smtp
```
Comment (akostadinov) [inline config/environments/development.rb:64]: This is to allow using actual SMTP delivery when SMTP configuration is present.

**Comment by hallelujah** [comment]
Nice PR, can you fix Code Climate? also and trigger the `javascript_tests` and `mysql_build` checks?

**Review by hallelujah — CHANGES_REQUESTED** [review]
Remark, in your comment you mentionned sidekiq but some emails are sent in the request so technically `rails server` is also impacted

**Comment by akostadinov** [comment]
hmm, I was thinking it's gonna complain if I use `var = case ...` but it got the other way around. I should have checked locally.

**Review by hallelujah — APPROVED** [review]

**Pull request merged**